src/Atmos/Model/AtmosModel.jl

```diff
@@ -312,16 +302,16 @@ function init_aux!(m::AtmosModel, aux::Vars, geom::LocalGeometry)
 end
 
 """
-    source!(m::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
+    source!(m::AtmosModel, source::Vars, state::Vars, aux::Vars, diffusive::Vars, t::Real)
```

Comment:
Can we keep the argument order consistent with `flux_diffusive!` (i.e. `diffusive` before `aux`)?

Reply:
Yes, I'll swap the order